src/service.ts

```diff
@@ -394,10 +385,20 @@ export class DebuggerService implements IDebugger, IDisposable {
     return reply.body.variables;
   }
 
+  /**
+   * Dump the content of a cell.
+   * @param code The source code to dump.
+   */
+  private async _dumpCell(
```

Comment:
This method is now private as it is only accessed from the service and is not part of the public `IDebugger` interface.